src/ozdisplay/SDL2misc.c

```diff
@@ -77,7 +59,9 @@ SDL_Surface *getsprite(char *filename) {
 
 /*****************   initsprite ***************************
  * requires square sprites layed out length-wise in spritesheet
- * returns number of sprites. Completes array of SDL_Rects for extraction
+ * can be any size (pixels) so long as it is square
+ * number of animations within sprite computed via sprite width/height
+ * returns number of animations. Completes array of SDL_Rects for extraction
  */
```

Comment:
That change is unrelated to the commit

Reply:
Yes - it maybe should have been a separate commit.    My lame excuse is that while inside the code, I couldn't remember how sprites worked, and was disgusted with my poor description so after significant work to re-figure sprites out, I changed the comments immediately.   The change was noted in the details of the  commit, but not the header....